code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -746,7 +746,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 //trying to insert or remove an id
 /obj/item/pda/proc/id_check(mob/user, obj/item/card/id/I)
 	if(!I)
-		if(id)
+		if(id && (src in usr.contents))

```

Comment:
User, don't use usr unless necessary. Unless that argument has different meaning here.

Reply:
I mixed it up with user. Just one time I don’t test and it all goes wrong